routes/login.py

```diff
@@ -0,0 +1,41 @@
+
+from flask import Blueprint, request, jsonify
+from flask_jwt_extended import create_access_token
+
+from models.user import User, ActiveType
+from util.error import Error
+
+Login = Blueprint('Login', __name__)
+
+
+@Login.route('/login', methods=['POST'])
+def login():
+    """Performs a login for the given user credentials.
+    Parameters:
+      String -- user: The user's name to login.
+      String -- password: The user's password in plaintext to login.
+    Returns:
+      200: If the login was successfull.
+      401: If either user name was not found or password does not match.
+      404: If the useraccount is not active.
+    """
+    j = request.get_json(force=True)
+    user_requested = j['user']
+    password_requested = j['password']
+
+    targetUsers = User.query.filter_by(name=user_requested).all()
```

Comment:
I don't really understand this. Why should there be more than one target user which matches the user name, given that the user name is unique as stated in models/user.py?